torchci/lib/bot/ciflowBot.ts

```diff
@@ -631,23 +248,7 @@ export class Ruleset {
       body += `\n| ${row[0]} | ${row[1].join(", ")} | ${row[2]} |`;
     }
 
-    body += `
-<br/>
-You can add a comment to the PR and tag @pytorchbot with the following commands:
-<br/>
-
-\`\`\`sh
-# ciflow rerun, "ciflow/default" will always be added automatically
-@pytorchbot ciflow rerun
-
-# ciflow rerun with additional labels "-l <ciflow/label_name>", which is equivalent to adding these labels manually and trigger the rerun
-@pytorchbot ciflow rerun -l ciflow/scheduled -l ciflow/slow
-\`\`\`
-
-<br/>
-
-For more information, please take a look at the [CI Flow Wiki](https://github.com/pytorch/pytorch/wiki/Continuous-Integration#using-ciflow).
```

Comment:
the wiki now is probably outdated, should we update it or do you think the ciflow comment is enough documentation?